tests/fixtures/directives.admonitions.md

```diff
@@ -0,0 +1,216 @@
+Admonition:
+.
+```{admonition} This is a **title**
+An example of an admonition with custom _title_.
+```
+.
+<aside class="admonition">
+<header class="admonition-title">This is a <strong>title</strong></header>
+<p>An example of an admonition with custom <em>title</em>.</p>
+</aside>
+.
+
+Note on split lines:
+.
+```{note} An example
+of an admonition on two lines.
+```
+.
+<aside class="admonition note">
+<header class="admonition-title">Note</header>
+<p>An example
+of an admonition on two lines.</p>
+</aside>
+.
+
+[FIX] Note on a single line [See #154](https://github.com/executablebooks/MyST-Parser/issues/154)
+.
+```{danger} An example of an admonition on a single line.
+```
+.
+<aside class="admonition danger">
+<header class="admonition-title">Danger</header>
+<p>An example of an admonition on a single line.</p>
+</aside>
+.
+
+Admonition with overridding class name
+.
+```{admonition} This is a title
+:class: tip
+An example of a `tip` with a custom _title_.
+```
+.
+<aside class="tip admonition">
+<header class="admonition-title">This is a title</header>
+<p>An example of a <code>tip</code> with a custom <em>title</em>.</p>
+</aside>
```

Comment:
All of these now put the custom class first, as is done in RST. I will make a note of that in the docs I am writing about this.